spec/TodoList.spec.js

```diff
@@ -1,144 +1,103 @@
-const TodoList = require("../src/TodoList.js")
+const TodoList = require('../src/TodoList.js')
 
-describe("TodoList", () => {
+describe('TodoList', () => {
   let todoList
 
   beforeEach(() => {
     todoList = new TodoList()
   })
 
-  it("creates a todo item", () => {
-    // set up
-    const expected = {
-      id: 1,
-      text: "turn the heating on!",
-      status: "incomplete"
-    }
-
-    // execute
-    const result = todoList.create("turn the heating on!")
-
-    // verify
-    expect(result).toEqual(expected)
+  // create(str)
+  it('creates a todo item', () => {
+    const result = todoList.create('turn the heating on!')
+    const expected = 'turn the heating on!'
+    expect(result.text).toEqual(expected)
   })
 
-  it("returns all items", () => {
-    // set up
-    const item1 = {
-      id: 1,
-      text: "turn the heating on!",
-      status: "incomplete"
-    }
-    const item2 = {
-      id: 2,
-      text: "Do the washing up",
-      status: "incomplete"
-    }
+  // showAll()
+  it('returns all items', () => {
+    const item1 = todoList.create('turn the heating on!')
+    const item2 = todoList.create('Do the washing up')
     const expected = [item1, item2]
 
-    // execute
-    todoList.create("turn the heating on!")
-    todoList.create("Do the washing up")
+    expect(todoList.showAll()).toEqual(expected)
```

Comment:
You should be verbose and explicit in your tests, try to hard code as much of the expected output as possible. All this test is doing now is checking that the `create` function returns something, you're not actually testing what it returns